docs/input.md

```diff
@@ -123,7 +123,7 @@ class TapableComponent extends PositionComponent with Tapable {
 
 Flame provides a simple way to access Flutter's features regarding accessing Keyboard input events.
 
-To use it just add the `KeyboardEvents` mixin to your game class. By doing so you will need to implement the `onKeyEvent` method, this method is called every time a keyboard event happens, and it receives an instance of the Flutter class `RawKeyEvent`, which can be used to get information about the event, like if it was a key down or key up event, the pressed key and etc.
+To use it just add the `KeyboardEvents` mixin to your game class. By doing so you will need to implement the `onKeyEvent` method, this method is called every time a keyboard event happens, and it receives an instance of the Flutter class `RawKeyEvent`. The event can be used to get information about what occurred, like if it was a key down or key up event, a pressed key etc.
```

Comment:
I believe that the original text specified that you got information about which key was pressed, not just that a key was pressed.

Reply:
The sentence is so long though, and has 5 commas. But I can try to restructure it so that it is closer to the original.

Reply:
The only thing that needs to be changed is this part "a pressed key etc.". To something like "which key was pressed, etc.".